kodilla-advanced-tests/src/test/java/com/kodilla/parametrized_tests/homework/PersonTest.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.parametrized_tests.homework;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PersonTest {
+    @ParameterizedTest
+    @MethodSource(value = "com.kodilla.parametrized_tests.homework.PersonSource#providePersonsFOrTestingBmi")
+    public void shouldCalculateBMI(double height, double weight, String text) {
+        Person person = new Person(height, weight);
+        assertEquals(text, person.getBMI());
```

Comment:
wiem ze nie bylo tego w Kodilli, ale dobra praktyka jest trzymanie sie konwencji
//Given //When //Then
lub 
//Arrange //Act //Assert

w takim zapisie to jest 
```
//Given
Person person = ...

//When
result = person.getBMI ..

//Then
assertEquals ...
```